recipes/mistune/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.7.2" %}
+
+package:
+  name: mistune
+  version: {{ version }}
+
+source:
+  fn: mistune-{{ version }}.tar.gz
+  url: https://github.com/lepture/mistune/archive/v{{ version }}.tar.gz
+  md5: 798a72534237c25e6349d24652d0a6bc
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - mistune
+
+about:
+  home: https://github.com/lepture/mistune
+  license:  BSD
```

Comment:
I usually add the info [here](https://github.com/lepture/mistune/blob/master/setup.py#L113) with the number of items [here](https://github.com/lepture/mistune/blob/master/LICENSE). (But some people get this really all messed up!)